integration-tests/paho-mqtt5/src/main/resources/application.properties

```diff
@@ -15,3 +15,6 @@
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
 quarkus.native.resources.includes=*.jks
+
+# Required by NetworkModuleService.setURIField
+quarkus.native.additional-build-args=-J--add-opens=java.base/java.net=ALL-UNNAMED
```

Comment:
Do we need to add some notes in docs of `paho-mqtt5`? is it only needed in ssl cases?

Reply:
I went back to look at it again. And the test is actually passing without it.

Maybe I previously had something misconfigured that triggered the problem code in `NetworkModuleService`.

I can revert the `--add-opens` and create follow up issue to investigate further as there's potentially an issue there for some scenarios (to be verified). WDYT?

Reply:
Sure, it looks good to me.